ci/infra/libvirt/terraform.tfvars.json.ci.example

```diff
@@ -1,14 +1,14 @@
 {
     "libvirt_uri": "qemu:///system",
+    "libvirt_keyfile": "",
```

Comment:
Do we need to add docs for this new var?

Reply:
docs? what are you having in mind? We do not have any libvirt variable documented AFAIK, however probably we should add it in `terraform.tfvars.example` file. 

Probably I can improve the README for libvirt to mention at least the variables configurable in `vars.yml`. Is something like that what are you imagining?

Reply:
What about now?

Reply:
:+1: